lib/fcm.js

```diff
@@ -106,7 +106,11 @@ FCM.prototype.send = function(payload, CB) {
 
                     // Success, return message id (without id=), or something error happened
                     if (id) resolve(id);
-                    if (error) reject(error);
+                    if (error) {
+                      const err = new Error();
+                      err.message = error;
+                      reject(err);

```

Comment:
nit. to be consistent, please change to 4-space indentation

Reply:
I will Change the Space indentation.